app/jobs/comments/calculate_score_job.rb

```diff
@@ -6,6 +6,7 @@ def perform(comment_id)
       comment = Comment.find_by(id: comment_id)
 
       comment&.update_columns(score: BlackBox.comment_quality_score(comment), spaminess_rating: BlackBox.calculate_spaminess(comment))
```

Comment:
Should we bail completely here if the comment doesn't exist with a guard clause?